--- run0edit/run0edit
+++ secureblue/files/system/usr/bin/run0edit
@@ -30,12 +30,12 @@
 
 from typing import Final, Sequence
 
-VERSION: Final[str] = "0.5.0"
+VERSION: Final[str] = "0.5.0-secureblue"
 
 
 def find_command(command: str) -> str | None:
     """Search for command using a default path."""
-    return shutil.which(command, path=os.defpath)
+    return f"/usr/bin/{command}"
 
 
 def editor_path(
@@ -148,7 +148,7 @@
 
 def find_command(command: str) -> str | None:
     """Search for command using a default path."""
-    return shutil.which(command, path=os.defpath)
+    return f"/usr/bin/{command}"
 
 
 def readonly_filesystem(path: str) -> bool | None:
